docs/user-guide/usage/node-api.md

```diff
@@ -58,6 +62,8 @@ Boolean. If `true`, at least one rule with an "error"-level severity registered
 
 A string displaying the formatted problems (using the default formatter or whichever you passed).
 
+If the `fix` option is set to `true`, a string displaying the autofixed code.
+
```

Comment:
```suggestion
A string that contains either the:

- formatted problems (using the default formatter or whichever you passed)
- or the autofixed code, if the `fix` option is set to `true`

```

Let's make it more explicit that it can contain one of two things.